src/routes/(app)/namespaces/[namespace]/workflows/[workflow]/[run]/user-metadata/+page.svelte

```diff
@@ -0,0 +1,49 @@
+<script>
+  import MetadataEvents from '$lib/components/workflow/metadata/metadata-events.svelte';
+  import WorkflowCurrentDetails from '$lib/components/workflow/metadata/workflow-current-details.svelte';
+  import WorkflowSummaryAndDetails from '$lib/components/workflow/metadata/workflow-summary-and-details.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { groupEvents } from '$lib/models/event-groups';
+  import { fullEventHistory } from '$lib/stores/events';
+  import { workflowRun } from '$lib/stores/workflow-run';
+
+  const { workflow } = $derived($workflowRun);
+
+  const relevantEvents = $derived(
+    $fullEventHistory.filter((event) => {
```

Comment:
Let's not filter the events quite yet. Let's create the event groups first, then filter the groups by group.userMetadata?.summary. That way we have the full status of the group. How it works now is that it filters it down to only Scheduled/Started events and then creates groups, which won't include the Completed event with the final status